app/decorators/keyword_decorator.rb

```diff
@@ -0,0 +1,17 @@
+# frozen_string_literal: true
+
+class KeywordDecorator < SimpleDelegator
```

Comment:
Is this decorator needed as it does not seem used?

Reply:
Removed in 6310a736cc744121a554172071b42970e95a8e83